modules/nanopb/CMakeLists.txt

```
@@ -0,0 +1,54 @@
+# Copyright (c) 2021, Basalte bv
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_NANOPB)
+
+  set(NANOPB_DIR ${ZEPHYR_CURRENT_MODULE_DIR})
+
+  include(${NANOPB_DIR}/extra/FindNanopb.cmake)
```

This gives a warning for CMake 3.17 and newer.

Please use:
```suggestion
  set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${NANOPB_DIR}/extra)
  find_package(Nanopb REQUIRED)
```

Note, warning seen:
```
CMake Warning (dev) at /opt/cmake-3.19.1-Linux-x86_64/share/cmake-3.19/Modules/FindPackageHandleStandardArgs.cmake:424 (message):
  The package name passed to `find_package_handle_standard_args` (Nanopb)
  does not match the name of the calling package (Zephyr).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  /projects/github/ncs/modules/lib/nanopb/extra/FindNanopb.cmake:365 (find_package_handle_standard_args)
  ../../../modules/nanopb/CMakeLists.txt:9 (include)
This warning is for project developers.  Use -Wno-dev to suppress it.
```